src/components/AddSenior.tsx

```diff
@@ -114,28 +121,42 @@ const AddSenior = ({
   seniorPatch,
   setSeniorPatch,
 }: AddSeniorProps) => {
-  const [seniorData, setSeniorData] = useState<SeniorData>(EMPTY_SENIOR);
   const [selectedStudents, setSelectedStudents] = useState<User[]>([]);
   const [currentImage, setCurrentImage] = useState<string | StaticImageData>(
     ImageIcon
   );
   const [confirm, setConfirm] = useState<boolean>(false);
   const [error, setError] = useState<boolean>(false);
+  const [edited, setEdited] = useState<boolean>(false);
```

Comment:
I think this may not be necessary if we still display the `save` button.

Reply:
Used to dim the save button now